keycloak.d.ts

```diff
@@ -18,12 +18,18 @@ interface KeycloakConnectStatic {
 declare namespace KeycloakConnect {
 
   interface KeycloakConfig {
-    'confidential-port': string|number
     'auth-server-url': string
-    'resource': string
-    'ssl-required': string
+    'realm': string
+    'clientId': string
+    'secret': string
+    'resource'?: string
+    'public-client'?: string
```

Comment:
Type should be changed into boolean here: https://github.com/keycloak/keycloak-nodejs-connect/blob/74a0d7fd45a74dfb2554cdd5e920d0ef641c684b/middleware/auth-utils/config.js#L117